Fidesnoella/Backend_blog
# A working version hosted on heroku.

### What this PR do:
- Fix the ploblem with Heroku platform
- Fix missing environments

### How to test:
Ensure that you have `.env` file in root directory with these arguments
```
MONGODB_URI="Link to your mongo atlas database"
TOKEN_SECRET="YoutTokenForJWT"
```
Then
```
npm install
npm run dev
```

### Links:
- Heroku: [Here](https://noella-atlp.herokuapp.com/)
